ServerCore/Properties/ServiceDependencies/puzzlehunt - Web Deploy/profile.arm.json

```diff
@@ -0,0 +1,113 @@
+{
```

Comment:
Extra file?

Reply:
Shoot - I've been pointedly not adding this file for many many consecutive PRs and finally forgot. Will remove!